nemo/collections/nlp/nm/trainables/common/megatron/megatron_bert_nm.py

```diff
@@ -124,8 +130,18 @@ def forward(self, input_ids, attention_mask, token_type_ids):
             input_ids, position_ids, extended_attention_mask, tokentype_ids=token_type_ids
         )
         return sequence_output
-
-    def restore_from(self, path, local_rank=0):
+    
+    def megatron_restore_from(self, path, local_rank=None): 
```

Comment:
maybe rename this one to smth like restore_model_parallel_megatron? also check that the new path exists? 
so for this to work, users should provide path to the checkpoint folder not the checkpoint .pt file, right? we need to make it clear in the training scripts args

Reply:
Yes, this is how I updated the checkpoint arg for token_classificaction.py:
`parser.add_argument(
    "--bert_checkpoint",
    default=None,
    type=str,
    help="Path to model file. (If using model parallel, path to checkpoint directory.)"
)`

Reply:
Updated the naming, and checking paths.